Examples/ObjectiveC/CalloutDelegateUsageExample.m

```diff
@@ -69,13 +69,16 @@ - (void)mapView:(MGLMapView *)mapView annotation:(id<MGLAnnotation>)annotation c
 {
     // Hide the callout view.
     [self.mapView deselectAnnotation:annotation animated:NO];
+    
+    // Create an alert containing the annotation's details
+    UIAlertController* alert = [UIAlertController alertControllerWithTitle:annotation.title
```

Comment:
Our house style (and the best style, obvs) is to put the pointer asterisk immediately in front of the object name.